### ISSUE TITLE
ganglia alternative port not working

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#232]
**Reported by efocht on 2 Aug 2006 12:37 UTC**
Reported by John Meskes:
```
> Has anyone tried entering an alternate port number to 8649 in Step 2 
> configure selected OSCAR packages, Ganglia.
> It changes the udp_send_channel on master, but nothing else, and no 
> difference is seen on the ganglia webpage. I then manually changed 
> /etc/gmond.conf udp_recv_channel port as well, plus changed 
> all 3 port references in that file on all the nodes, now the webpage 
> only shows this cluster.
```

Verified by Bernard Li.
Findings by Erich Focht:
  - post_rpm_nochroot exits early if no per-image config is chosen.

**Comment by dikim33** [comment]
**Comment by efocht on 2 Aug 2006 12:47 UTC**
Fixed in trunk svn r5267

**Comment by dikim33** [comment]
**Comment by efocht on 4 Aug 2006 15:07 UTC**
(In [5286]) Merged commit r5267 by efocht from 2006-08-02 14:46:12 +0200 (Wed, 02 Aug 2006)
with svnmerge_1by1 version v1.1 31.7.2006
from repository https://svn.oscar.openclustergroup.org/svn/oscar/trunk
to repository https://svn.oscar.openclustergroup.org/svn/oscar/branches/branch-5-0
Original commit message was:

Fix for bug #233: ganglia alternative port not working

Closes #233
